AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -229,6 +229,8 @@ newsweek.com##iframe[src="http://sns.europe.newsweek.com/newsletter?popup=1"]
 !######### JS tweaks ##########
 !##############################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/81316
+citychilli.com#%#//scriptlet("adjust-setInterval", "counter--", "", "0.02")
```

Comment:
@wout-codes I can't find where this rule should be applied

Reply:
This is inside the article.